components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1537,8 +1536,7 @@ func (p *pg) GetProject(ctx context.Context, id string) (*v2.Project, error) {
 	}
 
 	var project v2.Project
-	// Retrieve project if ID found AND intersection between projects and projectsFilter,
-	// unless the projectsFilter is empty (v2.0 case).
+	// Retrieve project if ID found AND intersection between projects and projectsFilter
```

Comment:
I am not following the comment because the ending is missing. (You did not break it, I realize.)
Is it supposed to conclude with "... is non-empty" ?
(Same in next block, too)